### PR TITLE
Added github profile finder project

### PR DESCRIPTION
**Pull request by Kaustav2410**
https://github.com/dhairyagothi/50_days_50_web_project/assets/94297897/33e8473f-af3b-47c7-b95d-741d31965e4d

### CODE REVIEW

**Comment by dhairyagothi** [comment]
@Kaustav2410 PULL main branch sync FORK

image

add as day 39

File: public/index.html [hunk 1]
```diff
@@ -216,6 +216,11 @@
             <div class="day-number">Day 37</div>
             <div class="project-name">2048 Game</div>
             <a href="2048_game/index.html" class="demo-link">Demo</a>
+         </div>
+          <div class="table">
+            <div class="day-number">Day 38</div>
```
Comment (dhairyagothi) [inline public/index.html:221]: Do it DAY 39

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

**Comment by dhairyagothi** [comment]
@Kaustav2410 
image

this project is made using which famework ?

**Comment by Kaustav2410** [comment]
Made using reactjs and tailwind the link i provided in the index.html is dummy link i saw other links which were referencing to the github repo so i wrote similar link since it is written in react can link the index.js file

**Comment by dhairyagothi** [comment]
> Made using reactjs and tailwind the link i provided in the index.html is dummy link i saw other links which were referencing to the github repo so i wrote similar link since it is written in react can link the index.js file

Ya ya got it 
I will merge the PR

**Comment by Kaustav2410** [comment]
thanks

**Comment by dhairyagothi** [comment]
@Kaustav2410 please drop email ID

**Comment by Kaustav2410** [comment]
kmajumder2002@gmail.com

**Pull request merged**